docs/modules/dht.md

```diff
@@ -10,6 +10,7 @@ Constants for various functions.
 
 ## dht.read()
 Read all kinds of DHT sensors, including DHT11, 21, 22, 33, 44 humidity temperature combo sensor.
+Return correct readout except for DHT12 and negative temperatures by DHT11.
```

Comment:
Maybe point to the specific `read` functions for those from here then?

Reply:
Like this?

Reply:
Yep, thanks.